app/presenters/detailed_guide_presenter.rb

```diff
@@ -43,6 +43,24 @@ def related_mainstream
     links("related_mainstream")
   end
 
+  def other
+    hash = {}
+    if !applies_to_all_nations
+      hash["Applies to"] = applies_to
```

Comment:
I've coalesced my thoughts in this commit: https://github.com/alphagov/govuk-content-schemas/pull/300/commits/55c6fef7af0850a5e7bdd4e8b0adaefee4e6ee18

In short, they're going to be send down translated, so nothing to do on this end.